tools/pfe-tools/react/generate-wrappers.ts

```diff
@@ -40,26 +40,39 @@ async function writeReactWrapper(
   if (!tagName) {
     throw new NonCriticalError(`declaration does not have a tag name: ${decl.name}`);
   } else {
+    const javascript = dedent;
+    const typescript = dedent;
```

Comment:
Do lines 43 / 44 do the same thing or are they different?

Reply:
they're just aliases for the sake of IDE syntax highlighting